### DISCUSSION TITLE
add Google application permission

### DISCUSSION THREAD

**Discussion by UmerSHK** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#653] [Ideas]
add Google application permission. So Google Assistant can run YouTube or show result via google

**Comment by sumeshir26** [comment]
@UmerSHK https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client#enabling-personal-results-optional

**Comment by im-coder-lg** [comment]
Don't forget to enable account linking!
